.ci/doc/templates/default.tpl.java

```diff
@@ -4,6 +4,7 @@
 import java.util.concurrent.ConcurrentHashMap;
 import io.kuzzle.sdk.coreClasses.responses.Response;
 import io.kuzzle.sdk.coreClasses.SearchResult;
+import io.kuzzle.sdk.coreClasses.lang.Lang;
```

Comment:
Isn't there any way we could beautify these paths :cry: `lang.Lang.lang.Lang`

Reply:
`lang` is the name of the package (the folder) and `Lang` the class. I don't know :/ how to beautify it :/
Thats like `io.kuzzle.sdk.coreClasses.responses.Response;`

Reply:
Yep, I know. Just hoping there could be a way to get better paths, but well.